src/test/kotlin/com/igorwojda/list/formattrainroute/challenge.kt

```diff
@@ -4,7 +4,20 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun formatTrainRoute(stations: List<String>): String {
-    TODO("not implemented")
+    var start="Train is calling at "
```

Comment:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/join-to-string.html is a very powerful tool

Reply:
var start="Train is calling at "
    if(stations.size==1){
        start+=stations.get(0)
    }else{
        start+=stations.joinToString { it }
        val index= start.lastIndexOf(",")
        start=start.substring(0,index)+" and"+start.substring(index+1)

    }
return start